player/src/main/java/com/tritondigital/player/MediaPlayer.java

```diff
@@ -684,12 +660,7 @@ final void setState(int state) {
         }
     }
 
-    final void notifyAnalytics(Format format) {
```

Comment:
Are some parts of the analytics removed?

Reply:
Yes, there isn't supposed to be any analytics in the public repo